### ISSUE TITLE
Secondary text color fails AA contrast requirements

### ISSUE THREAD

**Issue by Kataract** [CLEpy/clepy-website#17]
From aXe testing:
> Element has insufficient color contrast of 2.84 (foreground color: #999999, background color: #ffffff, font size: 10.5pt, font weight: normal). Expected contrast ratio of 4.5:1

Bumping the color from `#999` to `#6a6a6a` ups the ratio to 5.41:1 which passes AA.
